src/azul/service/catalog_controller.py

```diff
@@ -25,9 +27,8 @@ def list_catalogs(self) -> schema.object(
             additional_properties=schema.object(
                 atlas=str,
                 internal=bool,
-                plugins=schema.array(
-                    schema.object(
-                        name=str,
+                plugins=schema.object(
+                    name=schema.object(
```

Comment:
You can see the discrepancy here. This claims that the `plugins` object has a property called `name`.